### ISSUE TITLE
can't download or open from Novels Paradise

### ISSUE THREAD

**Issue by MJ933** [LNReader/lnreader-plugins#1101]
### Steps to reproduce

when i open novels paradise it show me an error 403

### Expected behavior

it should work correctly 

### Actual behavior

it not open to see and  download Novels 

### LNReader version

2.0.0 

### Android version

android 11

### Device

Xiaomi pad 5

### Other details

thank you for your time.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/lnreader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[2.0.0](https://github.com/lnreader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by error7404** [comment]
If you open the site in WebView you should see that you have been banned from the site

**Comment by MJ933** [comment]
How to solve this problem?

**Comment by 8L4CK-8L4Z3** [comment]
I have this problem too‚ but only webview is banned when opened in browser it works just fine

**Comment by Cohinem** [comment]
try changing the user agent. take it from your working browser.

**Comment by error7404** [comment]
the user agent doesn't seem to change anything for it

**Comment by 8L4CK-8L4Z3** [comment]
Yeah i had already tried several times

**Comment by Ahmedz10** [comment]
Has anyone found a solution to this problem?

**Comment by 8L4CK-8L4Z3** [comment]
Yes the solution that worked for me is uninstalling the plugin downgrading to beta 1 and reinstalling v1.1.1 and opening webview normally rn i could bypass cloudflare and ta da it's working

**Comment by error7404** [comment]
That didn't work for me unfortunately

**Comment by 8L4CK-8L4Z3** [comment]
I am using adguard app and it may have done something with my ip or something and this is a possible solution too

**Comment by error7404** [comment]
Already tried with Adguard, didn't change anything

**Comment by 8L4CK-8L4Z3** [comment]
I think it worth mentioning that i have created a clone of the app using lucky patcher now the clone have the problem but the original don't have it anymore 
The ordre is :
Installing the clone of v2b2 
Try to open novel paradise in webview in clone
Activate adguard
Downlgrade to b1 
Uninstall the plugin and reinstall it
Try to open novel paradise in webview in original

**Comment by K1ngfish3r** [comment]
Not sure what changed, but all my devices can access novels paradise now. Before it used to be just one phone

**Comment by error7404** [comment]
I still have the issue on my phone, so it's not magically fixed
